modules/claims_api/app/controllers/claims_api/v2/veteran_identifier_controller.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'evss/error_middleware'
+
+module ClaimsApi
+  module V2
+    class VeteranIdentifierController < ApplicationController
+      skip_before_action :authenticate, only: %i[show]
+      ICN_FOR_TEST_USER = '1012667145V762142'
+
+      def show
+        headers_to_validate = %w[Authorization X-VA-SSN X-VA-First-Name X-VA-Last-Name X-VA-Birth-Date]
+        validate_headers(headers_to_validate)
+
+        render json: { icn: ICN_FOR_TEST_USER }
```

Comment:
im concerned about having "icn" explicitly here. would rather is be just simply "id" in case we ever want/need to change that identifier in the future

Reply:
That's fair.  I'll switch it.